tests/RobotFramework/README.md

```diff
@@ -0,0 +1,248 @@
+5. Type the following command into the Terminal: `sudo apt update`
+6. Type the following command into the Terminal: `sudo apt install openvpn3`. This will finally install the OpenVPN 3 package
+
+#### 1.1.3. MacOS
+1. [Installation guide for OpenVPN Connect Client on macOS](https://openvpn.net/vpn-server-resources/installation-guide-for-openvpn-connect-client-on-macos/)
+
+### 1.2. Add OpenVPN Profile 
+1. Request a Profile file (*.ovpn)
+2. Open the OpenVPN Connect app and click plus.
+3. Click **Browse** and locate the previously downloaded OpenVPN profile.
+4. Select the profile in the file directory click **Open** in the file explorer.
+5. Click **Add** to import the OpenVPN profile.
+
+## 2. List of Devices available
+
+1. Raspberry Pi 4 - 192.168.1.4	(user: pass:) to be used only for executing 
+2. Raspberry Pi 4 - 192.168.1.110 (user:pi pass:thinedge)
+2. Raspberry Pi 3 - 192.168.1.120 (user:pi pass:thinedge)
+3. Raspberry Pi 4 - 192.168.1.130 (user:pi pass:thinedge)
+
+## 3. Run an .robot file
+
+
+To run an robot file ssh to 192.168.1.4 and use the following command structure:
+
+**PLEASE NOTE: This is example command**
+
+`robot -d \results --timestampoutputs --log build_install_rpi.html --report NONE --variable BUILD:844 --variable HOST:192.168.1.130 /thin-edge.io-fork/tests/RobotFramework/tasks/build_install_rpi.robot`
+
+
+## 4. List of Robot files
+### 4.1. Task Automations
+### **Installing specific SW Build**
+
+	File name: `build_install_rpi.robot`	
+	What is this task automation doing:
+	1. Defining the Device ID, structure is (ST'timestamp') (eg. ST01092022091654)
+	2. Checking the architecture in order to download the right SW
+	3. Setting the file name for download
+	4. Checking if thinedge is already installed on device 
+		4.1. if not the selected build version will be downloaded and installed in following order
+		1. Install Mosquitto
+		2. Install Libmosquitto1
+		3. Install Collectd-core
+		4. thin-edge.io installation
+		5. Install Tedge mapper
+		6. Install Tedge agent
+		7. Install Tedge apama plugin
+		8. Install tedge apt plugin
+		9. Install Tedge logfile request plugin
+		10. Install C8y plugin
+		11. Install Watchdog
+		12. Create self-signed certificate
+		13. Set c8y URL
+		14. Upload certificate 
+		15. Connect to c8y
+
+		4.2. if yes than thin-edge.io will be uninstalled using the uninstallation script to purge it and than steps from 4.1 will take place
+	
+	Example command to run the task: 
+	robot -d \results --timestampoutputs --log health_tedge_mapper.html --report NONE --variable HOST:192.168.1.110 health_tedge_mapper.robot
+
+### 4.2. Test Cases
```

Comment:
It's nice to have such list in README. On the other hand, you need to keep it in mind that you always need to update README when you add a new test file.